src/main/java/frc/team4909/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -0,0 +1,73 @@
+package frc.team4909.robot.subsystems.elevator;
+
+import edu.wpi.first.wpilibj.Encoder;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.command.InstantCommand;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+import java.lang.module.ModuleDescriptor.Requires;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import frc.team4909.robot.subsystems.elevator.SetElevatorPosition;
+import frc.team4909.robot.Robot;
+import frc.team4909.robot.RobotConstants;
+import frc.team4909.robot.operator.controllers.BionicF310;
+
+public class ElevatorSubsystem extends Subsystem{
+    WPI_VictorSPX VictorSPX1, VictorSPX2, VictorSPX3;
+    WPI_TalonSRX masterSRX;
+
+    public int holdingPosition;
+    public boolean encoderOverride;
```

Comment:
What is this variable? It is tested below, but never set to any particular value. If you're assuming it defaults to false, set it to false, but in any case, it appears it doesn't do anything. Then again, it's public. Is something outside of here modifying this variable? If so, there should be a setter met that sets it. 

Regardless, this requires either removing or documentation.